scripts/uberenv/packages/serac/package.py

```diff
@@ -249,6 +249,18 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
             path_replacements[tpl_root] = "${TPL_ROOT}"
             cfg.write(cmake_cache_entry("TPL_ROOT", tpl_root))
 
+        hypre_dir = get_spec_path(spec, "hypre", path_replacements)
+        cfg.write(cmake_cache_entry("HYPRE_DIR", hypre_dir))
+
+        metis_dir = get_spec_path(spec, "metis", path_replacements)
+        cfg.write(cmake_cache_entry("METIS_DIR", metis_dir))
+
+        parmetis_dir = get_spec_path(spec, "parmetis", path_replacements)
+        cfg.write(cmake_cache_entry("PARMETIS_DIR", parmetis_dir))
+
+        superludist_dir = get_spec_path(spec, "superlu-dist", path_replacements)
+        cfg.write(cmake_cache_entry("SUPERLUDIST_DIR", superludist_dir))
+
         mfem_dir = get_spec_path(spec, "mfem", path_replacements)
```

Comment:
How is tribol using serac host-config file?

Reply:
We are trying to get a tribol integration up and going as soon as possible.  So we are letting them use our TPL build until we are properly Spack'd on all required projects. This is a temporary hack.

Reply:
OK, living a TODO or a comment may be useful. Sometimes things last longer than we hope :)